web3j/src/main/java/com/vmware/ethereum/service/WorkloadCommand.java

```diff
@@ -46,13 +49,21 @@
   private final SecureTokenApi api;
   private final CountDownLatch countDownLatch;
   private final MetricsService metrics;
+  private final Web3jConfig web3jConfig;
+  public static Map<String, Instant> txHashTime = new HashMap<String, Instant>();
 
   @Override
   public void run() {
-    transferAsync();
+    if (web3jConfig.isQueuedPolling()) transferQueued();
```

Comment:
Use curly brace style if/else. We are going to enforce checkstyle later in the project to provide suggestion.

Reply:
done